pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -321,6 +321,7 @@ func (t TerraformDirectoryLoader) buildUnifiedConfig(rootMod *hclConfigs.Module,
 			// figure out path sub module directory, if it's remote then download it locally
 			var pathToModule string
 			var err error
+			var moduleDirDiags hcl.Diagnostics
```

Comment:
These can go in a single var block, just makes the code tidier

Reply:
I'll make this change .